worldedit-core/src/main/resources/lang/strings.json

```diff
@@ -491,6 +495,7 @@
 
     "worldedit.cyl.invalid-radius": "You must either specify 1 or 2 radius values.",
     "worldedit.cyl.created": "{0} blocks have been created.",
+    "worldedit.hcyl.thickness-too-large": "Thickness cannot be larger than x or z radii.",
```

Comment:
I started to name out additions with `fawe.`. Maybe we should continue doing so.